charts/sorry-cypress/templates/deployment-api.yml

```diff
@@ -63,6 +63,8 @@ spec:
           value: "mongodb://{{ include "mongodb.hostname" . }}:{{ .Values.mongodb.service.port }}"
         {{- end }}
         {{- end }}
+        - name: APOLLO_PLAYGROUND
+          value: {{ .Values.api.environmentVariables.apolloPlayground}}
```

Comment:
missing space before the `}}` I suspect this is what's breaking the CI.

Strongly recommend testing locally before pushing.

Reply:
Done